rustworkx/__init__.py

```diff
@@ -236,7 +236,7 @@ def adjacency_matrix(graph, weight_fn=None, default_weight=1.0, null_value=0.0):
 
         to return a weight of 1 for all edges. Also::
 
-            adjacency_matrix(graph, weight_fn: lambda x: float(x))
+            adjacency_matrix(graph, weight_fn: float)
```

Comment:
Both the old example and the new one are wrong by the way, it would be `adjacency_matrix(graph, weight_fn: lambda x:=float(x))`